### DISCUSSION TITLE
(balance) Calyxanide vs Calyx eggs makes no sense now

### DISCUSSION THREAD

**Discussion by CurePindal** [FakeFishGames/Barotrauma#5309] [Unstable feedback]
Hello! Now that 1 Calyx egg is crafted into just 1 Calyxanide (instead of 4 before) the shop prices of both makes no sense any longer.

Also the loot availability of the eggs vs the crafting change makes calyxanide significantly more rare.

Expected: 
1. Rebalanced prices of Calyxanide vs eggs.
2. Increased loot rates for eggs. 

thx for considering.

**Comment by Layarion** [comment]
Idk, 1 Calyx now seems to instantly remove 70% or more of husk infection. You used to NEED two usually, now 1 is a sure thing. so it might be fine.

**Reply by CurePindal** [reply]
> 
> 
> Idk, 1 Calyx now seems to instantly remove 70% or more of husk infection. You used to NEED two usually, now 1 is a sure thing. so it might be fine.

I think I was not referencing the efficiency of Calyx, but that the pricing makes no sense now.

huskeggs base price = 600
calyxanide base price = 250

1 huskegg is made into 1 calyxanide, so 600 degrades to 250. That was 600 to 1000 before.

With that values it makes no sense not to sell the eggs and buy calyx instead in most situations.
This is a bit sad, because buying raw materials so the medics have stuff to do, should be a crucial design goal.
With the change this values now undermine this. By a large margin.

**Reply by PaladynMinak** [reply]
You can use antibiotics, which are pretty cheap and easy to self obtain from environment, and I can imagine 'raw' eggs to be more valuable as potential weapon.

**Reply by adamkad1** [reply]
Still seems like a bad design to just sell eggs and buy calyx, makes doctors less useful. Also you sound like a husk cultist

**Reply by James-E-A** [reply]
I don't think it's a "bad thing" to have quasi-inconsistencies in pricing like this. As another very similar example, players might think deconstruct a swim bladder "to get stabilozine", but then actually thinking through it to realize that (if you can afford to wait on the STA until you're at an outpost) you can sell the bladder to buy more than you could make on your own: this is interesting, and an addition to the gameplay. Crafting is _not_ "automatically" better; like in real life, some things _aren't_ cheaper to DIY, and you've got to figure those out yourself.

Not to mention the fact that eggs are a far "stronger" item than distilled non-prophylactic antibiotic; they're a _weapon_. You may need 8 calyxanide through a biome, but just 2 eggs allows you to get up to _all kinds_ of shenanigans.

**Comment by Swato85** [comment]
I am with Pinda, its very strange for raw  materials to be priced above products, mainly when there islimited alternative usage and no overflow of product.
